build.sbt

```diff
@@ -77,6 +77,16 @@ lazy val noop = crossProject(JSPlatform, JVMPlatform)
     name := "log4cats-noop"
   )
 
+lazy val ce3 = crossProject(JSPlatform, JVMPlatform)
+  .settings(commonSettings)
+  .dependsOn(core)
+  .settings(
+    name := "log4cats-ce3",
+    libraryDependencies ++= Seq(
+      "org.typelevel" %%% "cats-effect" % catsEffectV
+    )
+  )
+
```

Comment:
doesn't core already depend on CE3?

Reply:
Probably we can move this to the `slf4j` module to don't create a new module. Speaking from a user experience it'd be handier - one wouldn't need to add a new dependency. WDYT?

Reply:
Happy to go with whatever you guys think is most appropriate - just didn't add it to either of the existing ones because:
- Core doesn't depend on CE3
- Isn't strictly speaking an `slf4j` concern

Reply:
With #678 we will be depending on cats-effect, and I see no other way to resolve the UUIDGen issue otherwise.

We'll have to track how the dependency on CE in core goes there as well.

Reply:
Now it depends on `cats-effect-std`.  This still introduces the `IO` dependency.